NuGet.Config

```diff
@@ -5,6 +5,7 @@
     <clear />
     <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
+    <add key="buildtools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
```

Comment:
We need to take this dependency for now @eerhardt Pushing the new msbuild package to cli-deps every update is too tedious. The right fix here is to get the msbuild packages on their own feed.